SonarQube.TeamBuild.PreProcessor/RulesetGenerator.cs

```diff
@@ -33,16 +44,32 @@ public void Generate(SonarWebService ws, string sonarProjectKey, string outputFi
             {
                 throw new ArgumentNullException("outputFilePath");
             }
-            
-            var qualityProfile = ws.GetQualityProfile(sonarProjectKey);
-            var activeRuleKeys = ws.GetActiveRuleKeys(qualityProfile);
+
+            IEnumerable<string> activeRuleKeys;
+            if (ws.GetInstalledPlugins().Contains(requiredPluginKey))
```

Comment:
FYI, this is a bit annoying: SonarQube web services continue to return the old VB.NET quality profile and rules if the plugin has been uninstalled. IMO this is bug that should be fixed in SonarQube, but meanwhile we need to work around it.